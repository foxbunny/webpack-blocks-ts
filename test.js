var {createConfig} = require('webpack-blocks');
var ts = require('./index');

const tsConf = options => createConfig([
  ts(options)
]);

test('default options', () => {
  expect(tsConf()).toMatchSnapshot();
});

test('add loader options', () => {
  expect(tsConf({
    silent: true
  })).toMatchSnapshot();
});