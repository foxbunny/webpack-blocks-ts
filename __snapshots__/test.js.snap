// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add loader options 1`] = `
Object {
  "module": Object {
    "rules": Array [
      Object {
        "loader": "ts-loader",
        "options": Object {
          "silent": true,
        },
        "test": /\\\\\\.\\(ts\\|tsx\\)\\$/,
      },
    ],
  },
  "plugins": Array [],
  "resolve": Object {
    "extensions": Array [
      ".ts",
      ".tsx",
    ],
  },
}
`;

exports[`default options 1`] = `
Object {
  "module": Object {
    "rules": Array [
      Object {
        "loader": "ts-loader",
        "options": undefined,
        "test": /\\\\\\.\\(ts\\|tsx\\)\\$/,
      },
    ],
  },
  "plugins": Array [],
  "resolve": Object {
    "extensions": Array [
      ".ts",
      ".tsx",
    ],
  },
}
`;
